src/extension/url-cache.js

```diff
@@ -34,6 +34,21 @@ export function isSharePointHost(url, projects = []) {
     });
 }
 
+/**
+ * Determines if a URL is a personal OneDrive folder.
+ * @param {string} url The tab URL
+ * @returns {boolean} {@code true} if personal OneDrive folder, else {@code false}
+ */
+export function isPersonalOneDriveFolder(url) {
+  const { pathname } = new URL(url);
+  return pathname.toLowerCase().endsWith('/onedrive.aspx')
+    || [
+      '/personal/',
```

Comment:
I think we should add `/my` too no?

Reply:
I would expect it to not open on this folder as well.

https://adobe-my.sharepoint.com/my?id=%2Fpersonal%2Fdyland%5Fadobe%5Fcom%2FDocuments%2Fhelix%2Dplayground